.github/workflows/test-action.yaml

```diff
@@ -35,8 +35,7 @@ jobs:
           key: ${{ runner.os }}-node-16-action-${{ hashFiles('yarn.lock') }}
       - name: Install dependencies
         run: |
-          cd action
-          yarn install
+          yarn install --frozen-lockfile
```

Comment:
Currently, the github action cache step does not work. This is because after `cd` into the directory, `yarn` will install local dependencies under `action`. Even though the `node_modules` exists under the root directory, it is not used.

Reply:
Looks like the cache still does not work.